README.md

```diff
@@ -531,11 +535,21 @@ Statics parameters
 - `jsCompilerWhitelist`: Whitelist of JS files to compile as an array. Leave undefined to compile all files. Supply a `:` character after each file name to delimit an alternate file path and/or file name for the minified file.
   - Default: `null` (compiles all JS files, if a JS minifier is enabled).
   - Example: *[String]* `library-name/example.js:lib/example.min.js` (compiles `library-name/example.js` into `lib/example.min.js`).
+  - `jsCompilerBlacklist`: Array of JS files to exempt from minification. These files will be copied as is to the build folder. Leave undefined to compile all files.
+  - Default: `null` (compiles all JS files, if a JS minifier is enabled).
+  - Example: *[String]* `example.js`
+
 - `jsCompiledOutput`: Where to place compiled JS files. This folder will be symlinked into `public` by default.
   - Default: *[String]* `".build/js"`.
+
 - `nodeEnv`: *[String]* Param to override the `NODE_ENV` environment variable.
   - Default: `undefined`.
 
+- `generateFolderStructure`: When enabled Roosevelt will generate user specified directories (e.g. mvc, statics, build).
```

Comment:
When enabled Roosevelt will generate user specified directories (e.g. MVC parameters and statics parameters).